src/Exttload.sol

```diff
@@ -0,0 +1,15 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.24;
```

Comment:
nit: I think most of ours we have `^0.8.24`. Do you have a strong feeling about this?

Reply:
`Extsload` is using `>=0.6.0` so I wanted to keep that convention but adjust it to a version where `tload` exists

Reply:
Yah makes sense